src/docker-compose-cli.js

```diff
@@ -31,7 +34,16 @@ const composeCommand = (filePaths, ...params) => {
   });
 };
 
-const pull = (fileName) => composeCommand(fileName, 'pull');
+const pull = async (fileName) => {
+  try {
+    await composeCommand(fileName, 'pull');
+  } catch (err) {
+    if (isRateExceededError(err)) {
+      return pull(fileName);
```

Comment:
Would it make sense to have a `setTimeout` here to let the endpoint cool off a bit before retrying?

Reply:
With the infinite retry, a timeout wasn't necessary. But once I added the limit of 100 retries, it became necessary - because the the rate is "per second".